localstack/services/awslambda/lambda_function.py

```diff
@@ -0,0 +1,13 @@
+class LambdaFunction(object):
```

Comment:
Can we please extend the existing `LambdaFunction` class here: https://github.com/localstack/localstack/blob/master/localstack/utils/aws/aws_models.py#L159
We can rename the `id` parameter to `arn` in the constructor of that that class.